### PR TITLE
Live ride fixes

### PR DESCRIPTION
**Pull request by guytepper**
- [resolved an issue where an route will change to another selected route if there's an active ride](7af6026959fe7df6ca024fc4381e995b1a9cd365)
- remove ride information bar on dynamic island expanded view
- remove bottom whitespace on lock screen view

### CODE REVIEW

File: ios/BetterRailWidget/Live Activity/Views/CountdownView.swift [hunk 1]
```diff
@@ -9,12 +9,10 @@ struct CountdownView: View {
   }
   
   var color: Color {
-    if (delay >= 5) {
-      return .red
```
Comment (planecore) [inline ios/BetterRailWidget/Live Activity/Views/CountdownView.swift:13]: I think we should keep it, the user need to be alerted that he may miss his connection.

File: ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift [hunk 2]
```diff
@@ -23,19 +23,18 @@ struct BetterRailLiveActivity: Widget {
       return DynamicIsland {
             // Expanded UI goes here.  Compose the expanded UI through
             // various regions, like leading/trailing/center/bottom
-          DynamicIslandExpandedRegion(.leading, priority: 90.0) {
+          DynamicIslandExpandedRegion(.leading) {
```
Comment (planecore) [inline ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift:26]: Why did you remove the priority?

File: ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift [hunk 3]
```diff
@@ -23,19 +23,18 @@ struct BetterRailLiveActivity: Widget {
...
-            }
-            DynamicIslandExpandedRegion(.bottom) {
-              RideInformationBar(vm: vm, placement: .dynamicIsland)
-            }
+            VStack {
+              Spacer()
+              TimeInformation(vm: vm, placement: .dynamicIsland)
+            }.padding(.trailing, 4)
+          }
+          DynamicIslandExpandedRegion(.bottom) {}
```
Comment (planecore) [inline ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift:37]: Why did you remove the ride bar? It has valuable information, even in the dynamic island.

File: ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift (hunk 2 above)
Comment (guytepper) [reply ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift:26]: It doesn't have an affect on the view anymore

File: ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift (hunk 3 above)
Comment (guytepper) [reply ios/BetterRailWidget/Live Activity/BetterRailLiveActivity.swift]: restored:
Screenshot 2023-05-20 at 4 22 40

**Review by planecore — APPROVED** [review]
